buildstockbatch/gcp/gcp.py

```diff
@@ -898,15 +991,17 @@ def main():
         elif args.postprocessonly:
             batch.build_image()
             batch.push_image()
-            batch.process_results()
+            batch.process_results(args.postprocesscloud)
         elif args.crawl:
             batch.process_results(skip_combine=True, use_dask_cluster=False)
         else:
             batch.build_image()
             batch.push_image()
             batch.run_batch()
-            batch.process_results()
-            batch.clean()
+            batch.process_results(args.postprocesscloud)
+            if not args.postprocesscloud:
+                # postprocesscloud is async, so don't want to clean before it's done
```

Comment:
Can we make this more consistent? So either we wait for the postprocessing to finish (similar to how we wait for the batch job) or we make both async (with a way to automatically trigger postprocessing when the simulations are ready)?

With this mix, users may or may not need to run the cleanup themselves, which feels unnecessarily confusing.

Reply:
Not sure; some thoughts...

* Post-processing is the last step, so that _could_ be a reason for it to behave differently (the batch kinda has to be synchronous so the script knows when to do post-processing, at least with the current overall implementation; but since the post-processing is the last step, it doesn't need to be).
* Post-processing doesn't report status (unlike Batch that has some ability to show progress, post-processing is just one job that's done or not), and post-processing takes longer according to some prior experience, so it makes less sense for the console to wait on it.

Yea, I think we should have a way to automatically trigger post-processing. Related, I've been thinking that more of the script should run on the cloud; like the weather data step, but also the thing controlling the steps (a means to "automatically trigger post-processing).

Let's figure that out, but I say we commit as-is in the meantime (and for that meantime, I appended "You will need to run the script with --clean to clean up GCP environment after post-processing is complete." to the "Cloud Run job started" output). wdyt?